### DISCUSSION TITLE
Enhancement: Apps and System use report

### DISCUSSION THREAD

**Discussion by MX10-AC2N** [IceWhaleTech/CasaOS#166] [Ideas]
Hi everyone,
I think it could be nice to have a visual of the use of the box on which our Casaos turns.
  - either directly from the web server.
  - either a daily mail sending, weekly or to be agreed.
   - either by messaging (matrix, or other).
I mean, of course, a return of the use of system resources and detailed by application that we would have installed and in use. And of course a return of the problems that could be when using apps.
I admit that personally I would like to receive this info directly on my mobile phone by a messaging like matrix without necessarily having to connect to my server.
This is an idea like any other, in hope that it can contribute to this beautiful creation that is Casaos.
What do you think about ?

**Comment by Joooost** [comment]
Great to have and see more insights from the UI! Could you provide more details of what you are looking for?

I'am using A Glances container with host permissions and it shows the host and container CPU, MEM, Network stats and more.

Alternative is netplan.io which also runs in a container and optional syncs with the netplan.io cloud service if you configured that.

Imho, there are alternatives but again to see more insights from de web UI is always welcome.

**Comment by JohnGuan** [comment]
This is a good idea, I will also be trying to learn as much data about my server in a Dashboard. Always gives me peace of mind.
But I don't seem to particularly like this kind of reporting.
What kind of information would you like to get in a weekly report?

PS: This's a good idea, best to format it again with reference to [guidelines ](https://github.com/IceWhaleTech/CasaOS/discussions/164)if you can.

**Comment by MX10-AC2N** [comment]
Hi @Joooost @JohnGuan
Thank you for your answers,
Hi everyone,
I was thinking something a bit like https://github.com/louislam/uptime-kuma
 I find this kind of monitoring very interesting and with this possibility of notification via messaging it's really the best I think..
Initially, I thought of connecting a mini-screen directly to the ZimaBoard in order to have a visual of the operation directly on my box, without needing to connect myself I had submitted this idea of ​​mini-screen on linked.in I imagined that to display a few things like Htop or Bashtop, but why not imagine a mini touch screen to display the monitoring and be able to directly interact with the operation of the various installed applications..
The top would even be able to define hours of operation for our different applications, we can define for example that nextcloud works during the day but stops in the evening and at night and conversely launched jellyfin at the end of the afternoon and stop it at middle of the night..

**Reply by Joooost** [reply]
Yeah that's great, especially the notification and integrations with Telegram and other push notification services.

**Reply by AuthorShin** [reply]
Check out this two project:
https://hackaday.io/project/181320-gnat-stats-tiny-oled-pc-performance-monitor
https://hackaday.io/project/19018-phat-stats-pc-performance-tft-display

**Reply by JohnGuan** [reply]
> Check out this two project: 
> https://hackaday.io/project/181320-gnat-stats-tiny-oled-pc-performance-monitor 
> https://hackaday.io/project/19018-phat-stats-pc-performance-tft-display

This is so COOL!

**Reply by MX10-AC2N** [reply]
Thank @AuthorShin 
His two projects are excellent, it's exactly this kind of small screen that would be essential on the zimaboard, during my exchanges on linked.in with Lauren Pan he showed me this https://www.seeedstudio.com/Wio-Terminal-p-4509.html
I hope that there will be a possible integration of this kind of screen (of monitoring).
Thanks again for sharing..

**Reply by AuthorShin** [reply]
Your welcome guys
I am not really sure of a way on debian OS to extract and broadcast real time system data but if there is a way for that it's not that hard to build up a standalone device with ESP32 which got internal BT and WiFi also support external battery.
So here is your portable Zimaboard (or any other device and server) monitor that connects over WiFi so you can have your Zimaboard inside your server room or closet and still monitor it. :)
